content/en/docs/Writing policies/match-exclude.md

```diff
@@ -15,6 +15,15 @@ The `match` and `exclude` clauses have the same structure and can each contain t
 
 At least one element must be specified in a `match` or `exclude` block. The `kind` attribute is optional when working with the `resources` element, but if it's not specified the policy rule will only be applicable to metadata that is common across all resource kinds.
 
+In addition, a user may specify the `group` and `apiVersion` with a kind in the match / exclude declarations for a policy rule.
+
+Supported formats:
+
+* `Group/Version/Kind`
+* `Version/Kind`
+* `Kind`
+
```

Comment:
@vyankyGH, could you add a couple examples of the G/V/K and V/K options here since these are new? This would help users understand how it should be formatted and also why it would be necessary over just specifying kind.